fe2-android/app/src/main/java/com/github/dedis/popstellar/ui/detail/LaoDetailActivity.java

```diff
@@ -291,6 +294,10 @@ private void setupSocialMediaActivity() {
             });
   }
 
+  private void setupDigitalCashMainActivity() {
+    // TODO do the things here when activity is created
+  }
+
```

Comment:
If the content of this function will be addressed in another PR I would remove it here since it does nothing. I think having empty functions is a bad practice